app/src/main/java/com/android/template/injection/fragment/FragmentComponentInjects.java

```diff
@@ -1,5 +1,11 @@
 package com.android.template.injection.fragment;
 
+import com.android.template.ui.welcome.WelcomeFragment;
```

Comment:
Ajde pogledam ti merge request danas do kraja dana. Ako ćeš raditi dalje na feature-ima, onda napravi branch iz ovog i nastavi sa radom.

Reply:
Ma samo polako, jos na ovom imam dosta za napraviti. To sam stavio request cisto da pogledas kad stignes